### PR TITLE
Import existing key material for CredentialBundle

### LINKED ISSUE #461: [Feature request] Support importing external `private_key` for `CredentialBundle`
**Is your feature request related to a problem? Please describe.**

Currently there is no way to pass an external `private_key` during creating of an `CredentialBundle` instance which does not allow for protocols where the private key is also used for features outside of MLS.

**Describe the solution you'd like**

An additional constructor in `CredentialBundle` which allows me to pass on a `private_key` via `SignatureKeypair` as a method argument.

**Additional context**

https://github.com/openmls/openmls/blob/main/openmls/src/credentials/mod.rs#L159

### PR DESCRIPTION
**Pull request by raphaelrobert**
This is a follow-up to #557. This PR adds functions to import existing keys for `CredentialBundle`.
Fixes #461.

### CODE REVIEW

**Comment by adzialocha** [comment]
I think the problem is this line here: https://github.com/openmls/openmls/blob/30fbec303bb228bd280714eed03e531899d475b5/openmls/src/ciphersuite/mod.rs#L43 :+1:

**Comment by adzialocha** [comment]
Also related to this: How do I create a new `SignaturePrivateKey` instance from an external secret key? `SignaturePublicKey` provides a way (`::new(bytes, signature_scheme)`) which is convenient.

**Comment by franziskuskiefer** [comment]
This PR needs

- [ ] a feature flag as proposed in #596 
- [ ] fix the prelude export (I recommend removing it from there. It's nothing that is commonly used.)

> How do I create a new `SignaturePrivateKey` instance from an external secret key?

@adzialocha why do you need to create a `SignaturePrivateKey`? `SignatureKeypair::from_bytes` as introduced in this PR allows you to get a key pair from raw bytes. There shouldn't be the need to create only the private key.

**Comment by adzialocha** [comment]
> why do you need to create a SignaturePrivateKey? SignatureKeypair::from_bytes as introduced in this PR allows you to get a key pair from raw bytes. There shouldn't be the need to create only the private key.

Yes, this can be considered as resolved now, at the point of my question it was not possible. So, alles gut 🖖

And thank you!

File: openmls/src/ciphersuite/signature.rs [hunk 1]
```diff
@@ -137,6 +149,10 @@ impl SignaturePublicKey {
             .map_err(|_| CryptoError::InvalidSignature)
     }
 
+    /// Get the signature scheme of the keypair
```
Comment (kkohbrok) [inline openmls/src/ciphersuite/signature.rs:152]:
```suggestion
    /// Get the signature scheme of the public key
```

File: openmls/src/ciphersuite/signature.rs [hunk 2]
```diff
@@ -159,4 +175,9 @@ impl SignaturePrivateKey {
             Err(_) => Err(CryptoError::CryptoLibraryError),
         }
     }
+
+    /// Get the signature scheme of the keypair
```
Comment (kkohbrok) [inline openmls/src/ciphersuite/signature.rs:179]:
```suggestion
    /// Get the signature scheme of the private key
```

**Review by kkohbrok — COMMENTED** [review]
Looks good! Just two nits.

**Review by franziskuskiefer — APPROVED** [review]

**Pull request merged**